frontend/language/src/en.json

```diff
@@ -73,9 +74,9 @@
   "dashboard.data_modelling_format_xsd": "XSD",
   "dashboard.data_models": "Data models",
   "dashboard.description": "Description",
-  "dashboard.edit_app": "Edit the app {{appName}}",
-  "dashboard.edit_data_models": "Edit models",
-  "dashboard.edit_service": "Edit app",
+  "dashboard.edit_app": "Edit {{appName}} in Studio",
+  "dashboard.edit_datamodels": "Edit {{appName}} in Studio",
```

Comment:
`"dashboard.edit_datamodels"` should be; `"dashboard.edit_data_models"` 
 Did it not work with playwright? 🫨

Reply:
This was just an error on my side when resolving merge conflicts 😅 Fixed ✅